src/onboarding2/Templates/TemplateDetails.js

```diff
@@ -200,7 +207,49 @@ function TemplateDetails({ template, visible, onUse, onClose }) {
               css={`
                 height: 150px;
               `}
-            />
+            >
+              {template.optionalModules.map(({ appName, label }, index) => (
+                <div
+                  key={index}
+                  css={`
+                    display: flex;
+                    justify-content: space-between;
+                    margin-top: ${2 * GU}px;
+
+                    & + & {
+                      margin-top: ${1.5 * GU}px;
+                    }
+                  `}
+                >
+                  <KnownAppBadge appName={appName} label={label} />
+                  <div
+                    css={`
+                      display: flex;
+                      align-items: center;
+                      ${unselectable}
+                    `}
+                  >
+                    <Checkbox
+                      checked={templateOptionalModules[appName]}
+                      onChange={() => {
```

Comment:
@bpierre I don't know if there's a way to avoid this function declaration without going too far out of our way. I think it's fine like this?

Reply:
Yes totally fine, we are almost at the last level of the tree +1